### PR TITLE
add slider value 

### LINKED ISSUE #914: wrong positon of resize-button.
screencapture-localhost-8080-2022-04-11-11_02_53 (1)

### PR DESCRIPTION
**Pull request by alsturm**
fixes #914

### CODE REVIEW

File: src/modules/menu/components/mainMenu/MainMenu.js [hunk 1]
```diff
@@ -121,6 +121,11 @@ export class MainMenu extends MvuElement {
 			container.style.width = parseInt(event.target.value) + 'em';
 		};
 
+		const getValue = () => {
+			const container = this.shadowRoot.getElementById('mainmenu');
+			return (container && container.style.width !== '') ? parseInt(container.style.width) : 28;
```
Comment (taulinger) [inline src/modules/menu/components/mainMenu/MainMenu.js:126]: seems to be a magic number... 
let's move it to a static getter and add a test on this method (see SWIPE_DELTA_PX for an example)
```
static get INITIAL_WIDTH_EM() {
		return 28;
	}
```

**Review by taulinger — COMMENTED** [review]
Nice, would just suggest removing the magic number by a named constant

File: src/modules/menu/components/mainMenu/MainMenu.js (hunk 1 above)
Comment (alsturm) [reply src/modules/menu/components/mainMenu/MainMenu.js]: done

**Review by taulinger — APPROVED** [review]
Nice!

**Pull request merged**